backend/apis/flag-utils/is-first-degree-felony-flag-util.js

```diff
@@ -0,0 +1,12 @@
+exports.isFirstDegreeFelony = function isFirstDegreeFelony(parsedObj) {
+  if (
```

Comment:
Do we need to iterate through "charges" if there is more than 1 charge on the docket? May need to add a for loop to do this?

Reply:
You can iterate through the `charges` array using `map` function. You can use the map method to return a new array that with only the value of the `severity` key and see if the the new array includes string "1st degree felony" or "amended".

Something like this:
```js
parseObj.charges.map(charge => charge.severity.toLowerCase()).includes("1st degree felony" || "amended")
```

Reply:
Contrived example but hopefully helps understand the usage of [`Array.map`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) here:

```js
const people = [{name: 'charley', age: 12}, {name: 'xavier', age: 23}, {name: 'tina', age: 11}]

const onlyNames = people.map(person => person.name)
//["charley", "xander", "luna"]

onlyNames.includes("charley" || "tina")
//true
```

Reply:
Thanks for the help! I think I have it now.